test/integration/connectors/test_qdrant.py

```diff
@@ -0,0 +1,82 @@
+import json
+import uuid
+from pathlib import Path
+from typing import Literal
+
+import pytest
+from qdrant_client import AsyncQdrantClient
+
+from test.integration.connectors.utils.constants import DESTINATION_TAG, env_setup_path
+from test.integration.connectors.utils.docker_compose import docker_compose_context
+from unstructured_ingest.v2.interfaces.file_data import FileData, SourceIdentifiers
+from unstructured_ingest.v2.processes.connectors.qdrant import (
+    CONNECTOR_TYPE,
+    QdrantConnectionConfig,
+    QdrantUploader,
+    QdrantUploaderConfig,
+    QdrantUploadStager,
+    QdrantUploadStagerConfig,
+)
+
+SERVER_URL = "http://localhost:6333"
+LOCAL_DIRECTORY_NAME = "qdrant"
+COLLECTION_NAME = uuid.uuid4().hex
+DOCKER_COMPOSE_DIR = env_setup_path / "qdrant"
+VECTORS_CONFIG = {"size": 384, "distance": "Cosine"}
+
+
+@pytest.mark.asyncio
+@pytest.mark.tags(CONNECTOR_TYPE, DESTINATION_TAG)
+@pytest.mark.parametrize("mode", ["remote", "local"])
+async def test_qdrant_destination(
+    upload_file: Path, tmp_path: Path, mode: Literal["remote", "local"]
+):
+    connection_kwargs = (
+        {"location": SERVER_URL}
+        if mode == "remote"
+        else {"path": str(tmp_path / LOCAL_DIRECTORY_NAME)}
+    )
+
+    with open(upload_file) as file:
+        elements = json.load(file)
+        EXPECTED_POINT_COUNT = len(elements)
+        EXPECTED_TEXT = elements[0]["text"]
+        EMBEDDINGS = elements[0]["embeddings"]
+
+    with docker_compose_context(DOCKER_COMPOSE_DIR):
```

Comment:
Do you need docker compose running when this is using local mode? If the test approach is so different, might make sense to have this as two separate tests rather than a single one that's parameterized.